karton/config_extractor/config_extractor.py

```diff
@@ -83,6 +83,10 @@ def args_parser(cls):
             default=[],
             nargs="+",
         )
+        parser.add_argument(
+            "--identity",
+            help="Override the default karton identity",
```

Comment:
```suggestion
            help="Override the default Karton identity",
```
Capitilized version is used in docstring